src/components/header/header.jsx

```diff
@@ -36,6 +43,17 @@ const Header = (props) => {
                   </Link>
                 }
               </li>
+              {isAuthorized &&
+                <li className="header__nav-item" style={{marginLeft: `10px`}}>
+                  <button className="button" onClick={handleLogoutClick}>
+                    <svg xmlns="http://www.w3.org/2000/svg" width={20} height={20} viewBox="0 0 24 24" fill="none" stroke="#777" strokeWidth={2} strokeLinecap="round" strokeLinejoin="round">
+                      <path d="M9 21H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h4"/>
+                      <polyline points="16 17 21 12 16 7"/>
+                      <line x1={21} y1={12} x2={9} y2={12}/>
+                    </svg>
```

Comment:
можно вынести в отдельный файлик и создать компоненту для иконок
```suggestion
  const Icon = (props) => (
                    <svg xmlns="http://www.w3.org/2000/svg" width={props.width} height={props.height} viewBox="0 0 24 24" fill="none" stroke="#777" strokeWidth={2} strokeLinecap="round" strokeLinejoin="round">
                      <path d="M9 21H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h4"/>
                      <polyline points="16 17 21 12 16 7"/>
                      <line x1={21} y1={12} x2={9} y2={12}/>
                    </svg>
  );
```